shell/platform/embedder/embedder_thread_host.cc

```diff
@@ -94,34 +116,86 @@ EmbedderThreadHost::CreateEmbedderManagedThreadHost(
     return nullptr;
   }
 
-  const auto platform_task_runner = CreateEmbedderTaskRunner(
+  // The UI and IO threads are always created by the engine and the embedder has
+  // no opportunity to specify task runners for the same.
+  // TODO(chinmaygarde): If we ever decide to expose more task runners to engine
```

Comment:
Consider using github issue ids for TODOs instead of personal IDs, since they might be confusing for future readers (is the person going to implement? are they the person to ask questions? is this in their current priorities? etc). 

In this case I don't see much value in using either, seems like it could just be part of the comment (but still avoid "we" ) :)

Reply:
Done.